.github/pull_request_template.md

```diff
@@ -1,13 +1,25 @@
 <!--
 
-Describe in detail the changes you are proposing, and the rationale.
+Describe in detail the changes you are proposing, and the rationale. If practical, describe how to manually test the change; this isn't always possible, but it makes PRs easier and faster to review.
 
 See the contributing guide:
 
 https://github.com/hashicorp/terraform/blob/main/.github/CONTRIBUTING.md
 
 -->
 
+## Reason for change
+
+## Old behavior
+
+## New behavior
```

Comment:
I have two related concerns:

1. Adding all these fields makes the PR template pretty long and could be a potential barrier to someone submitting a PR, especially if it's something simple.
2. Our contributing guide already sets the standard that [changes should be proposed in a GitHub issue first](https://github.com/hashicorp/terraform/blob/main/.github/CONTRIBUTING.md#proposing-a-change), so that would be the right place to describe this information, and not in the PR itself.

Thankfully both of these concerns can be addressed by a single solution: emphasize that a GitHub issue is the place to include this info, which simplifies the PR template quite a bit.

```suggestion
```